src/dmd/func.d

```diff
@@ -22,6 +22,7 @@ import dmd.dclass;
 import dmd.declaration;
 import dmd.delegatize;
 import dmd.dinterpret;
+import dmd.dmangle;
```

Comment:
Doesn't seem used anywhere ?

Reply:
Indeed. Most likely the original patch had the semantic pass in this module.  Now it's been moved to dsymbolsem.